chapters/install/ide-speedup.md

```diff
@@ -1,4 +1,4 @@
-# Speed-up Jina app development in PyCharm and VSCode
+# Speeding Up Development in PyCharm and VSCode
```

Comment:
This should be Title Case

Reply:
It already is

https://titlecase.com/

![Screenshot from 2021-05-17 09-54-21](https://user-images.githubusercontent.com/4182659/118451790-e5963000-b6f5-11eb-80bd-00df499eca12.png)